### ISSUE TITLE
adding cellTypes: e.g. 'D1 expressing neurons'

### ISSUE THREAD

**Issue by lzehl** [HumanBrainProject/openMINDS_controlledTerms#89]
@tgbugs : would it make sense to register neuron types such as 'D1 expressing neurons' or would you rather register this separately (e.g., in a study target field) as "neuron" and "D1 receptor" ?

**Comment by UlrikeS91** [comment]
I think we need to add this information on some level. If the receptor/protein information is too detailed (as is the case for 'D1 expressing neurons'), we could use e.g. 'dopaminergic neuron' instead.  
However, the information has an impact on the function of a neuron. To me, different subcellular composition would qualify as different cell types. 
It's as usual a matter of perspective and what granularity level we are aiming for. We should not aim for 'only shape' (e.g., 'basket cell', 'pyramidal cell', etc.), this would not include overarching functional types that should be captured (e.g., 'interneuron', 'place cell'). As stated before, a good compromise could be terms like 'dopaminergic neuron', 'inhibitory neuron', etc. They give some more details about the function, but not down to the receptor/protein level. 

Personally, I would still prefer cell types at the receptor/protein level, though.

**Comment by lzehl** [comment]
'dopaminergic neuron' is not the same as 'D1 expressing neurons' apparently. 'dopaminergic neuron' exists also already in the ontology but not for the receptor expressing neuron.

I agree with you that this level is detailed but highly valuable I think.

**Comment by UlrikeS91** [comment]
> 'dopaminergic neuron' is not the same as 'D1 expressing neurons' apparently. 'dopaminergic neuron' exists also already in the ontology but not for the receptor expressing neuron.
> 
> I agree with you that this level is detailed but highly valuable I think.

Yeah, true. Dopaminergic neurons (http://uri.interlex.org/base/ilx_0103395) are the ones releasing dopamine. D1 expressing neurons are the ones that have dopamine 1 receptors. Sorry, about that confusion...

**Comment by lzehl** [comment]
So my feeling goes towards registering "dopaminergic neuron" as well as "D1 expressing neuron". 
@UlrikeS91 and @tgbugs your feelings?

**Comment by tgbugs** [comment]
D1 dopamine receptor expressing neurons would be good to register. I think we need to add that to NPO.

**Comment by UlrikeS91** [comment]
also related to #94

**Comment by UlrikeS91** [comment]
conclusion: 
- add "**D1 dopamine receptor expressing neuron**" with synonyms: D1 receptor expressing neuron | D1R expressing neuron | D1 expressing neuron | DRD1 expressing neuron | dopamine receptor D1 expressing neuron; no ontology ID
- add "**dopaminergic neuron**" (http://uri.interlex.org/ilx_0103395)

**Comment by lzehl** [comment]
@UlrikeS91 I suggest to reduce the name for us to "D1 receptor expressing neuron"

**Comment by UlrikeS91** [comment]
> @UlrikeS91 I suggest to reduce the name for us to "D1 receptor expressing neuron"

Works for me, but I named it "D1 receptor expressing **cell**" to be consistent within the "<<protein>> expressing cells". All of these always got the "<<protein>> expressing neuron" as synonym and additionally a bunch of others (e.g., including abbreviation of the protein and such). "D1 dopamine receptor expressing neuron" has been added as a synonym in this particular case.
